resources/js/layouts/legal-layout.tsx

```
@@ -0,0 +1,47 @@
+import AppLogoIcon from '@/components/app-logo-icon';
```

ESLint

[error] 'AppLogoIcon' is defined but never used.

(`@typescript-eslint/no-unused-vars`)

---

[error] 'Button' is defined but never used.

(`@typescript-eslint/no-unused-vars`)

---

[error] 'login' is defined but never used.

(`@typescript-eslint/no-unused-vars`)

---

[error] 'create' is defined but never used.

(`@typescript-eslint/no-unused-vars`)

---

[error] `react` type import should occur before import of `@/components/app-logo-icon`

(import/order)